Cargo.toml

```diff
@@ -204,6 +204,10 @@ name = "cargo-example-derive"
 required-features = ["derive", "color"]
 doc-scrape-examples = true
 
+[[example]]
+name = "markdown-derive"
+required-features = ["derive"]
```

Comment:
We can remove this example or incorporate it in others once it's finished, was more for testing while developing.